src/mono/sample/wasi/http-p2/Program.cs

```diff
@@ -70,6 +72,23 @@ public static async Task<int> MainAsync(string[] args)
         Console.WriteLine();
         Console.WriteLine(json);
 
+        // test trailers
+        using (HttpClient httpClient = new HttpClient())
+        {
+            httpClient.DefaultRequestVersion = HttpVersion.Version20;
+            httpClient.DefaultRequestHeaders.Add("TE", "trailers");
```

Comment:
Understandably the line with `HttpVersion.Version20` doesn't do anything right now.

But I'm getting `ITypes.HeaderError.FORBIDDEN` from wasmtime v24 for setting `TE` header. Am I doing it wrong ?
I remeber there was some discussion about it, but can't find it now.

@dicej